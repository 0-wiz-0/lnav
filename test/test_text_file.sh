#! /bin/bash

export YES_COLOR=1
unset XDG_CONFIG_HOME

run_cap_test ${lnav_test} -n \
    ${top_srcdir}/README.md

run_cap_test ${lnav_test} -n \
    ${top_srcdir}/src/log_level.cc
