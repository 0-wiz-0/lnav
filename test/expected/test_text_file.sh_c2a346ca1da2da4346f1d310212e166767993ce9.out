[
    {
        "top_meta": {
            "file": "{top_srcdir}/README.md",
            "breadcrumbs": [
                {
                    "display_value": "README.md",
                    "search_placeholder": "",
                    "possibilities": [
                        {
                            "display_value": "README.md"
                        }
                    ]
                },
                {
                    "display_value": "LNAV – The Logfile Navigator",
                    "search_placeholder": "",
                    "possibilities": [
                        {
                            "display_value": "LNAV – The Logfile Navigator"
                        }
                    ]
                },
                {
                    "display_value": "Contributing",
                    "search_placeholder": "",
                    "possibilities": [
                        {
                            "display_value": "Contributing"
                        },
                        {
                            "display_value": "Features"
                        },
                        {
                            "display_value": "Installation"
                        },
                        {
                            "display_value": "Links"
                        },
                        {
                            "display_value": "Screenshot"
                        },
                        {
                            "display_value": "See Also"
                        },
                        {
                            "display_value": "Usage"
                        }
                    ]
                },
                {
                    "display_value": "⋯",
                    "search_placeholder": "",
                    "possibilities": [
                        {
                            "display_value": "Building From Source"
                        }
                    ]
                }
            ]
        }
    }
]
